typescript/vro-scripting-api/src/api/ConfigurationElement.ts

```diff
@@ -50,6 +52,7 @@ namespace vroapi {
          */
         setAttributeWithKey(key: string, value: any, typeHint?: any): void {
             let attr = this.getAttributeWithKey(key);
+            this._attributes = [];
```

Comment:
Wouldn't this overwrite the attributes every time?

Reply:
You're correct, that line was added before when testing, it has been reoved